singer/src/main/java/com/pinterest/singer/writer/kafka/CommittableKafkaWriter.java

```diff
@@ -138,20 +148,20 @@ public void startCommit() throws LogStreamWriterException {
       }
     }
 
-    commitableBuckets = new HashMap<>();
-    committableMapOfTrackedMessageMaps = new HashMap<>();
-    committableMapOfInvalidMessageMaps = new HashMap<>();
-    committableMapOfOriginalIndexWithinBucket = new HashMap<>();
-
+    committableBuckets.clear();
+    committableMapOfTrackedMessageMaps.clear();
+    committableMapOfInvalidMessageMaps.clear();
+    committableMapOfOriginalIndexWithinBucket.clear();
 
-    for (int i = 0; i < committableValidPartitions.size(); i++) {
+    for (PartitionInfo partitionInfo : committableValidPartitions) {
       // for each partitionId, there is a corresponding bucket in buckets and a
       // corresponding headersMap in mapOfHeadersMaps.
-      PartitionInfo partitionInfo = committableValidPartitions.get(i);
       int partitionId = partitionInfo.partition();
-      commitableBuckets.put(partitionId, new KafkaWritingTaskFuture(partitionInfo));
-      committableMapOfTrackedMessageMaps.put(partitionId, new HashMap<Integer, LoggingAuditHeaders>());
-      committableMapOfInvalidMessageMaps.put(partitionId, new HashMap<Integer, LoggingAuditHeaders>());
+      committableBuckets.put(partitionId, new KafkaWritingTaskFuture(partitionInfo));
```

Comment:
Why was there an indentation update?

Reply:
It was actually just fixing the `commitableBuckets` typo to `committableBuckets`